src/views/project/project/modules/project-alert/ProjectEscalationPolicy.vue

```diff
@@ -28,6 +28,24 @@
                                 {{ $t('PROJECT.DETAIL.ALERT.MINUTE') }}
                             </span>
                         </template>
+                        <div v-if="channelFormatter(rule.notification_level).length > 0" class="channel-box">
+                            <div v-for="(channel, cIdx) in channelFormatter(rule.notification_level)" :key="`channel-${cIdx}`"
+                                 :class="{ disabled: channel.state === CHANNEL_STATE.DISABLED }"
+                            >
+                                <p class="title">
+                                    [{{ CHANNEL_SCHEMA[channel.schema] }}] {{ channel.name }}
+                                    <p-i name="ic_bell" color="inherit" class="ml-1"
+                                         width="1rem" height="1rem"
+                                    />
+                                    {{ channel.state === CHANNEL_STATE.ENABLED ? 'ON' : 'OFF' }}
+                                </p>
+                                <div v-if="channel.schema === 'spaceone_user'" class="info">
```

Comment:
뒤에서 온 데이터.. 신뢰가 가지않아서 ..ㅋㅋㅋ
```suggestion
                                <div v-if="channel.schema === 'spaceone_user' && channel.data && Array.isArray(channel.data.users)" class="info">
```
최대한 방어해보면 어떨까요!

Reply:
감사요ㅎㅎ 추가했습니다